includes/Core/Permissions/Permissions.php

```diff
@@ -260,4 +260,24 @@ private function grant_additional_caps( array $allcaps ) {
 
 		return $allcaps;
 	}
+
+	/**
+	 * Get all capabilities used in Google Site Kit.
+	 *
+	 * @since 1.x
```

Comment:
Our workflow for new code is to use `n.e.x.t`, can you please change this accordingly?
```suggestion
	 * @since n.e.x.t
```